image/Dockerfile

```diff
@@ -26,23 +31,38 @@ RUN make build-linux
 
 # --------------------------------------------------------------------------------------------------
 # The downloading stage.
-FROM alpine:latest AS downloader
+FROM ${DOWNLOADER_IMAGE} AS downloader
 
-ENV HELM_VERSION="v2.13.1"
-ENV FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
+ARG HELM_VERSION="v2.16.7"
+ARG HELM_FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
 
-RUN wget -O /tmp/${FILENAME} http://storage.googleapis.com/kubernetes-helm/${FILENAME}
-RUN tar -zxvf /tmp/${FILENAME} -C /tmp
+RUN wget -O /tmp/${HELM_FILENAME} http://storage.googleapis.com/kubernetes-helm/${HELM_FILENAME}
+RUN tar -zxvf /tmp/${HELM_FILENAME} -C /tmp
 
 # --------------------------------------------------------------------------------------------------
-# The running stage.
-FROM debian:stretch
+# The cert building stage.
+FROM ${CERT_BUILDER_IMAGE} AS cert_builder
+
+ARG CURL_VERSION="7.70.0"
+
+RUN zypper refresh
+RUN zypper --non-interactive install perl-Encode make tar gzip curl
```

Comment:
If we're going to do all this, might as well just install `ca-certificates` and grab `/var/lib/ca-certificates/ca-bundle.pem` (or `/etc/ssl/ca-bundle.pem` which is a symlink to that file) out of it? It's not any _worse_, as far as I can tell…

Reply:
I guess it's another way of doing the same thing. I don't think one is better than the other... I may just keep as-is.